litmus-portal/frontend/public/icons/github.svg

```diff
@@ -0,0 +1,10 @@
+<svg width="15" height="15" viewBox="0 0 15 15" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
What purpose does this file change serve?

Reply:
I have added this earlier for UI purpose